process-cli/src/main/assembly/filter-dist.properties

```diff
@@ -0,0 +1 @@
+use_embedded_jre=true
```

Comment:
What is the use of this?

Reply:
Good catch, it's a leftover we don't need. (Was supposed to serve as a property we could query from launcher scripts). Removed in [79ddfa3](https://github.com/SonarSource/sonarlint-core/pull/757/commits/79ddfa394bd9a8f78543ac932e2eda5e69cdc53b)